docs/ref/release_notes.rst

```diff
@@ -1,6 +1,117 @@
 Release Notes
 =============
 
+=======
+What's new in iris-grib v0.16
+-----------------------------
+
+:Release: 0.16.0
+:Date: ?? Sep 2020
+
+Features
+^^^^^^^^
+
+* `@tpowellmeto <https://github.com/tpowellmeto>`_ added support for loading
+  data on a "Lambert Azimuthal Equal Area Projection",
+  i.e. grid definition template 3.140.
+  See `PR#187 <https://github.com/SciTools/iris-grib/pull/187>`_ .
+
+* `@bjlittle <https://github.com/bjlittle>`_ made all the tests runnable for a
+  packaged install of iris-grib, where the grib testdata files will be missing.
+  See `PR#212 <https://github.com/SciTools/iris-grib/pull/212>`_ .
+
+* `@m1dr <https://github.com/m1dr>`_ added support for loading statistical
+  fields, as encoded in production definition template 3.8, even when the
+  "interval time increment" value is not specified (i.e. set to "missing").
+  See `PR#206 <https://github.com/SciTools/iris-grib/pull/206>`_ .
+
+* `@pp-mo <https://github.com/pp-mo>`_ ported some tests from Iris, which test
+  grib saving of data loaded from other formats.
+  See `PR#213 <https://github.com/SciTools/iris-grib/pull/213>`_ .
+  All grib-dependent testing is now contained in iris-grib : **There are no
+  remaining tests in Iris which use grib.**
```

Comment:
PR reference should be at the end

Reply:
It isn't really about this PR, just this one happened to be the last outstanding.  
I might just add it as a separate bullet.